### ISSUE TITLE
Bot will not use skills no errors just will not click skills

### ISSUE THREAD

**Issue by illmiken** [kolton/d2bot-with-kolbot#2366]
My bot runs fine no errors i can see it even switch to skills it is supposed to be using but does not use them it will just stand in front of the boss while my merc attacks. 

Here is my attack config but i dont believe it is wrong. Am i missing something stupid that the bot will not use it's skills?

Config.AttackSkill[0] = -1; // Preattack skill. Not implemented yet.
	Config.AttackSkill[1] = 56; // Primary skill to bosses.
	Config.AttackSkill[2] = 47; // Primary untimed skill to bosses. Keep at -1 if Config.AttackSkill[1] is untimed skill.
	Config.AttackSkill[3] = 56; // Primary skill to others.
	Config.AttackSkill[4] = 47; // Primary untimed skill to bosses. Keep at -1 if Config.AttackSkill[3] is untimed skill.
	Config.AttackSkill[5] = 64; // Secondary skill if monster is immune to primary.
	Config.AttackSkill[6] = 64; // Secondary untimed skill if monster is immune to primary untimed.

**Comment by DarkHorseDre** [comment]
Is the boss fire and cold immune?

Do you have 'TeleStomp' enabled?

**Comment by illmiken** [comment]
> Is the boss fire and cold immune?
> 
> Do you have 'TeleStomp' enabled?

the boss is not cold immune it does it on every boss

i do have tele stomp enabled i will try to turn that off to see what happens.

**Comment by DarkHorseDre** [comment]
It shouldnt use telestomp if boss is not dual immune IIRC - plus you will see it telestomp

what does the console (press home key) say when this happens?

**Comment by illmiken** [comment]
> It shouldnt use telestomp if boss is not dual immune IIRC - plus you will see it telestomp
> 
> what does the console (press home key) say when this happens?

it says control not found 6 594 433 location null

**Comment by DarkHorseDre** [comment]
No worries

Hmm I've had that but not sure what it is (didn't happen often)

it's created by oog.js - not sure how it works.
```js
print("control not found " + type + " " + x + " " + y + " location " + getLocation());
```
with 6 being type 
x=594
y=433

if your bot is not standard or being interfered with I'd download a separate copy of kolbot - copy your char profile, profiles.json and keys and see if you can reproduce the issue on base code.

Hopefully someone else can help - you should try searching existing issues on the "control not found" text too

**Comment by illmiken** [comment]
> No worries
> 
> Hmm I've had that but not sure what it is (didn't happen often)
> 
> it's created by oog.js - not sure how it works.
> 
> ```js
> print("control not found " + type + " " + x + " " + y + " location " + getLocation());
> ```
> 
> with 6 being type
> x=594
> y=433
> 
> if your bot is not standard or being interfered with I'd download a separate copy of kolbot - copy your char profile, profiles.json and keys and see if you can reproduce the issue on base code.
> 
> Hopefully someone else can help - you should try searching existing issues on the "control not found" text too

so i just finished downloading the everything again and starting fresh i am getting zero error codes this time but my bot is still not using any of their skills not sure what i am missing.

**Comment by DarkHorseDre** [comment]
depends what you've set in profile (post to pastebin and link here)
[check the custom attack and immune attack configs]

did it work correctly before with that character (and before any stat reset)?

if you cant find the issue, quickest thing to do is create a new config file with just the attack config changed to see if it attacks
(assuming you have meteor, fireball and forb skills with hard points as configured)

**Comment by illmiken** [comment]
here is my profile
![Capture](https://user-images.githubusercontent.com/63060245/78296991-b1b17880-74fc-11ea-8ed7-3d479ac26823.PNG)

here is my complete config
[jNXaJuRX.txt](https://github.com/kolton/d2bot-with-kolbot/files/4423378/jNXaJuRX.txt)

**Comment by DarkHorseDre** [comment]
i meant character config - delete those images plx and paste your char config to pastebin.com (so I get syntax highlighting via brownser and dont need to play with your files)

but check everything I asked in my last post as it may save time

**Comment by illmiken** [comment]
[jNXaJuRX.txt](https://github.com/kolton/d2bot-with-kolbot/files/4424040/jNXaJuRX.txt)

**Comment by DarkHorseDre** [comment]
You keep attaching the file to github - what you will need to do lots on this forum (and any forum where you need to paste lots of text to get help) is paste to pastebin.com, save the file then paste the link.
here's one I did earlier: https://pastebin.com/Yb4LrgBH - notice the domain of my link

anyway, I had a quick look at your file - this isnt the file you were using when you had the issue is it? it only has andariel set and not much else..

**Comment by illmiken** [comment]
> You keep attaching the file to github - what you will need to do lots on this forum (and any forum where you need to paste lots of text to get help) is paste to pastebin.com, save the file then paste the link.
> here's one I did earlier: https://pastebin.com/Yb4LrgBH - notice the domain of my link
> 
> anyway, I had a quick look at your file - this isnt the file you were using when you had the issue is it? it only has andariel set and not much else..

yes this is the file im using i redid it and started from scratch i kept it very simple to take any variables on why it is not working

**Comment by DarkHorseDre** [comment]
>... you should try searching existing issues on the "control not found" text too

seems you didnt bother to search bro. have a read

https://github.com/kolton/d2bot-with-kolbot/issues?q=is%3Aissue+%22control+not+found%22

**Comment by illmiken** [comment]
> > ... you should try searching existing issues on the "control not found" text too
> 
> seems you didnt bother to search bro. have a read
> 
> https://github.com/kolton/d2bot-with-kolbot/issues?q=is%3Aissue+%22control+not+found%22

i read all of those dark thanks though. the error i was getting was from clicking create game before the bots delay i told you i didnt have any errors. i did figure out my issue though thanks for your help it was none of those.
